docs/internal/releasing.md

```diff
@@ -41,5 +41,8 @@ git push
 
 6. Update documentation:
 * Edit CHANGELOG.md with release notes
-* Update getting_started.md with new version
+* Update `stable` tag
+```
+git tag stable --force && git push upstream --tags --force
```

Comment:
Safer to do:
```
git tag stable --force && git push upstream stable --force
```
Otherwise you may propagate other tags unintentionally.

Reply:
sounds good. changed